### PR TITLE
LPS-171655 Help Texts for remaining TBA

### PR DESCRIPTION
**Pull request by oliv-yu**
https://issues.liferay.com/browse/LPS-171655 - [FE] Help Texts for remaining TBA

Added remaining helptexts, which includes subtitles for the basic/blueprint options inside Suggestion Contributor of searchbar configuration. Also updated wording in system settings suggestions configurator configuration 👍 

Screen Shot 2023-01-06 at 11 30 33 AM
(This image mocks the learn link, but links should render after the CDN server updates the JSON for portal-search-web.)

### CODE REVIEW

**Comment by oliv-yu** [comment]
ci:test:sf

**Comment by oliv-yu** [comment]
ci:test:search

**Review by thektan — CHANGES_REQUESTED** [review]

File: modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/components/SearchBarConfigurationSuggestions.js [hunk 1]
```diff
@@ -419,16 +421,41 @@ function Inputs({onChange, onReplace, contributorOptions, value = {}}) {
...
+							<ClayButton
+								aria-label={Liferay.Language.get(
+									'suggestion-contributor'
+								)}
+								className="form-control form-control-select"
+								displayType="unstyled"
+							>
+								{value.contributorName === CONTRIBUTORS.BASIC
+									? Liferay.Language.get('basic')
+									: Liferay.Language.get('blueprint')}
```
[the remark this answers is not in the document]
Comment (oliv-yu) [reply modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/components/SearchBarConfigurationSuggestions.js]: Added a commit to use `contributorOptions` instead! 👍

**Review by thektan — APPROVED** [review]
Thanks for the update! Changes look good! 💯

**Comment by thektan** [comment]
ci:test:sf

**Comment by thektan** [comment]
ci:test:search

**Comment by brookedalton** [comment]
Continued in https://github.com/liferay-search/liferay-portal/pull/736

**Pull request closed without merging**